webapp/app.py

```diff
@@ -618,20 +601,16 @@ async def on_prepare(request, response):  # pylint: disable=unused-argument
 
     app.app.on_response_prepare.append(on_prepare)
     app.app.router.add_get("/metrics", metrics)
+    app.app.router.add_get("/healthz", HealthHandler.get)
```

Comment:
why not use `healthz.spec.yaml` in webapp but use in reposcan?

Reply:
I wanted to use healthz spec for both, the problem is that the AIOHTTP app in webapp does not allow having empty api prefix, because connexion is building the AIOHTTP app through `add_subapp` function.

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/connexion/apps/aiohttp_app.py", line 59, in add_api
    self.app.add_subapp(api.base_path, api.subapp)
  File "/usr/local/lib64/python3.6/site-packages/aiohttp/web_app.py", line 302, in add_subapp
    raise ValueError("Prefix cannot be empty")
ValueError: Prefix cannot be empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 21, in <module>
    "": "healthz.spec.yaml"})
  File "/git/app.py", line 612, in create_app
    arguments={'vmaas_version': VMAAS_VERSION})
  File "/usr/local/lib/python3.6/site-packages/connexion/apps/aiohttp_app.py", line 62, in add_api
    "aiohttp doesn't allow to set empty base_path ('/'), "
connexion.exceptions.ConnexionException: aiohttp doesn't allow to set empty base_path ('/'), use non-empty instead, e.g /api
```

Reply:
ok, nm